pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -936,13 +936,19 @@ const (
 	TaintNodeCriticalComponentsNotReady = "node.gardener.cloud/critical-components-not-ready"
 	// LabelNodeCriticalComponent is the label key for marking node-critical component pods.
 	LabelNodeCriticalComponent = "node.gardener.cloud/critical-component"
+	// LabelMachineName defines the labels which contains the name of the machine of a node
+	LabelMachineName = "node.gardener.cloud/machine-name"
```

Comment:
I did it the same way as for `node.gardener.cloud/critical-components-not-ready` which is also in [MCM](https://github.com/gardener/machine-controller-manager/blob/db952c97b572c311f9b1b1abe2723206d9961476/pkg/util/provider/machineutils/utils.go#L65) and [gardener](https://github.com/gardener/gardener/blob/e66b38b47d3504ac8f9de4b5215863fd3d807caf/pkg/apis/core/v1beta1/constants/types_constants.go#L917) repositories.
It duplicates the constant, but the labels are probably better documented, because most people check the `gardener/gardener` repository. I don't have a strong opinion here.

Reply:
Let's not duplicate the constant if we can easily import it from another package. I don't know why it was done like this for `node.gardener.cloud/critical-components-not-ready`. Let's clean this up as well.